guideforms.py

```diff
@@ -335,6 +335,20 @@
         ('If your feature was available as an Origin Trial, link to a summary '
          'of usage and developer feedback. If not, leave this empty.')),
 
+    'i2e_lgtms': forms.EmailField(
+        required=False, label='i2e LGTM by',
+        widget=forms.EmailInput(
+            attrs={'multiple': True, 'placeholder': 'email'}),
+        help_text=('Full email address of API owner who LGTM\'d the '
+                   'Intent to Experiment email thread.')),
+
+    'i2s_lgtms': forms.EmailField(
+        required=False, label='i2s LGTMs by',
+        widget=forms.EmailInput(
+            attrs={'multiple': True, 'placeholder': 'email, email, email'}),
+        help_text=('Full email addresses of API owner who LGTM\'d '
```

Comment:
ownerS.

(Also, does this need to say comma-delimited?)

Reply:
Done.